test/listener_test.py

```diff
@@ -137,13 +135,16 @@ def test_filetype_to_instance(extension, klass):
         test_filetype_to_instance(".log", can.CanutilsLogWriter)
         test_filetype_to_instance(".txt", can.Printer)
 
-        # test file extensions that should use a fallback
-        test_filetype_to_instance("", can.Printer)
-        test_filetype_to_instance(".", can.Printer)
-        test_filetype_to_instance(".some_unknown_extention_42", can.Printer)
         with can.Logger(None) as logger:
             self.assertIsInstance(logger, can.Printer)
 
+        # test file extensions that should use a fallback
```

Comment:
Can you update/remove this comment since we're changing the default behaviour? With your changes, there is no longer a fallback to the `Printer`, and instead we just raise the `ValueError` from the `Logger`'s `__init__` function.

Reply:
Sure, must've forgotten that one.